website/client/src/components/payments/selectUserModal.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <b-modal
+    id="select-user-modal"
+    :hide-header="true"
+    :hide-footer="true"
+    @hide="onHide()"
+  >
+    <div class="g1g1 mb-4 text-center">
+      <div class="heading">Gift a Subscription, Get a Subscription</div>
```

Comment:
do we want to use translatable strings here?

Reply:
Yeah, this was sort of a placeholder because we're not actually running the promo right now. I can go ahead and get the strings in for translation and do things proper-like, though!